gems/sorbet-runtime/README.md

```diff
@@ -6,6 +6,32 @@ Full docs are at:
 - https://sorbet.org/docs/runtime
 - https://sorbet.org/docs/tconfiguration
 
+## Environment variables
```

Comment:
Unlike with the `sorbet` gem, we have managed to keep all the docs for sorbet-runtime on sorbet.org to this point. I would love if we could move the docs for these environment variables to the docs in tconfiguration.md

Reply:
Opened #6679 to add docs.